examples/send-and-receive-emails/send-and-receive-emails-1-smtp.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/email;
+import ballerina/io;
+
+public function main() {
+    // Creates an SMTP client with the connection parameters, host, username, and password.
+    // Default port number `465` is used over SSL with these configurations.
+    email:SmtpClient smtpClient = new ("smtp.email.com", "sender@email.com", "pass123");
```

Comment:
IINM we need to limit the column length up to 80 to avoid jumping to the next line during the BBE generation for the website. Therefore shall we structure these lengthy lines into two. This is not applicable for the comments

Reply:
Structured limiting the line length to 80 characters.